data/news/sonoma.json

```diff
@@ -3,6 +3,26 @@
   "title": "Sonoma County COVID-19 News",
   "home_page_url": "https://sonomacounty.ca.gov/News/",
   "items": [
+    {
+      "id": "https://sonomacounty.ca.gov/CDC/News/COVID-19-Rental-Assistance/",
+      "url": "https://sonomacounty.ca.gov/CDC/News/COVID-19-Rental-Assistance/",
+      "title": "COVID-19 Rental Assistance\u00a0For Sonoma County Renters and Landlords",
+      "date_published": "2020-10-14T00:00:00-07:00",
+      "summary": "Renter households having trouble paying rent or utilities due to COVID-19 job loss or reduced hours; have a COVID-19 positive exposure or diagnosis.* Landlords who have tenants behind on rent due to COVID-19 job loss or reduced hours; or have a COVID-19 positive exposure or diagnosis.",
+      "tags": [
+        "Sonoma CountyCommunity Development Commission"
```

Comment:
Need to look into whether this is correctly parsing tags. The lack of space here makes me wonder if the page structure has changed. (Not worth holding up merging for.)